### ISSUE TITLE
Modifier keys aren't working (in Wayland?)

### ISSUE THREAD

**Issue by ghost** [SebastianLuebke/improved-osk-gnome-ext#2]
Hey, thanks for picking up where this extension was before. I believe this problem was also in the old extension, but for some reason, keys like Ctrl aren't working under Wayland, so things like Ctrl+C in terminal only output "c". In XWayland apps like Chromium they work, however.

Sorry for any trouble, I just love this extension as I have a tablet and the default onscreen keyboard in GNOME is just really bad, lol.

**Comment by luebke-dev** [comment]
Yeah, you're right. Seems like there are multiple bugs with Wayland. On my desktop the keyboard doesn't even show up automatically when pointing on something in chromium or other programs.

I will add this to my todo list.

**Comment by luebke-dev** [comment]
I switched to Wayland now, I will try to fix that soon
